packages/modals/src/styled/StyledHeader.spec.tsx

```diff
@@ -13,12 +13,12 @@ describe('StyledHeader', () => {
   it('renders default styling', () => {
     const { container } = render(<StyledHeader />);
 
-    expect(container.firstChild).toHaveClass('c-dialog__header');
+    expect(container.firstChild).toHaveStyleRule('color', '#2f3941');
```

Comment:
Should these values be calculated from the `DEFAULT_THEME` or `PALETTE` objects?

Reply:
In another PR, @jzempel had mentioned to me to avoid calculating CSS units using `DEFAULT_THEME`'s space.base unit in spec files. As a result, I hardcoded the color value here.

I could update this assertion to look directly at the `DEFAULT_THEME`:

```jsx
expect(container.firstChild).toHaveStyleRule('color', DEFAULT_THEME.palette.red[600]);
```

Or it could use the `getColor` function to derive the color from the `DEFAULT_THEME`:

```jsx
expect(container.firstChild).toHaveStyleRule(
  'color',
  getColor('dangerHue', 600, DEFAULT_THEME)
);
```

Do ya'll have a preference?

Reply:
Oh, didn't realize that was the direction we were going now. If that's the case feel free to resolve!

My only comment is that hard-coding these values makes these start to feel like snapshot tests rather than unit/integration tests.